### DISCUSSION TITLE
curl: (35) error:0A000126:SSL routines::unexpected eof while reading

### DISCUSSION THREAD

**Discussion by RemareX** [Cyfrin/foundry-full-course-cu#1295] [General]
Hi, I keep getting this error when trying to install foundry on wsl. How to fix please.

This is the command i used 
"curl -L https://foundry.paradigm.xyz | bash"

**Comment by WaleedMaghraby** [comment]
use a VPN, 
I faced this error and tries all solutions suggested
VPN works great

**Comment by WaleedMaghraby** [comment]
you can also try this command:
 curl -k -L https://foundry.paradigm.xyz | bash
